<launch>

  <arg name="robot_pose_x" value="4.4"/>
  <arg name="robot_pose_y" value="1.6"/>
  <arg name="robot_pose_Y" value="1.0"/>

  # 환경 시뮬레이션
  <include file="$(find shared_control)/launch/gazebo.launch">
    <arg name="robot_pose_x" value="$(arg robot_pose_x)"/>
    <arg name="robot_pose_y" value="$(arg robot_pose_y)"/>
    <arg name="robot_pose_Y" value="$(arg robot_pose_Y)"/>
  </include>

  # 이동로봇 제어
  <node pkg="joy" type="joy_node" name="joy_node"/>
  <node pkg="shared_control" type="direct_controller.py" name="direct_controller" output="screen">
    <param name="robot_vel_lin" value="0.26"/>
    <param name="robot_vel_ang" value="1.0"/>
    <param name="freq" value="0.1"/>
  </node>

  # 인터페이스
  <node pkg="rviz" type="rviz" name="rviz"
        args="-d $(find shared_control)/rviz/direct_control.rviz"/>

</launch>
